docs/extensionAPI/vscode-api-commands.md

```diff
@@ -136,17 +136,6 @@ let success = await commands.executeCommand('vscode.previewHtml', uri);
 
 `vscode.previewHtml` - Render the html of the resource in an editor view.
 
-Links contained in the document will be handled by VS Code whereby it supports `file`-resources and [virtual](https://github.com/Microsoft/vscode/blob/master/src/vs/vscode.d.ts#L3295)-resources as well as triggering commands using the `command`-scheme. Use the query part of a command-uri to pass along JSON-encoded arguments - note that URL-encoding must be applied.
```

Comment:
This section all lives in a comment right now. I'm in favor of consolidating it all under the new html preview section